src/uu/seq/src/floatparse.rs

```diff
@@ -0,0 +1,176 @@
+use crate::extendedbigdecimal::ExtendedBigDecimal;
+use crate::number::PreciseNumber;
+use crate::numberparse::ParseNumberError;
+use bigdecimal::BigDecimal;
+use num_traits::FromPrimitive;
+
+pub fn parse_hexadecimal_float(s: &str) -> Result<PreciseNumber, ParseNumberError> {
```

Comment:
Maybe these functions can be used by other coreutils programs, no?

Reply:
Perhaps. Since I'm not very familiar with the other modules yet, I’ve only used them in seq so far. But I’m sure we can make them shared if needed